internal/configs/version1/nginx-plus.tmpl

```diff
@@ -54,8 +54,18 @@ http {
                       '"$http_user_agent" "$http_x_forwarded_for"';
     {{- end}}
 
-    {{- if and .AppProtectDosLoadModule (.AppProtectDosLogFormat)}}
-    log_format log_dos '{{.AppProtectDosLogFormat}}';
+    {{- if .AppProtectDosLoadModule}}
+        {{- if .AppProtectDosLogFormat}}
```

Comment:
could you fix the indentation here and in line 68?

Reply:
done